src/components/Home/Hero.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import React from "react";
+import bikeWoman1 from "../../assets/images/bike-woman 1.png";
+
+const Hero = () => (
+    <div className="flex flex-col md:flex-row sm:flex-row">
+        <div>
+            <Image src={bikeWoman1} className="h-1 w-1" width={200} height={200} alt="..." />
```

Comment:
You'll want to put some descriptive text in the `alt` even if the image is just decorative it's good for accessibility purposes